### PR TITLE
Applying Carol's JSON API doc changes

### PR DESCRIPTION
**Pull request by leo-da**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/json-api/index.rst [hunk 1]
```diff
@@ -316,7 +314,7 @@ Examples
 Create a new Contract
 *********************
 
-See the request documentation below on how to create an instance of ``Iou`` contract from the :doc:`Quickstart guide </app-dev/bindings-java/quickstart>`:
+See the request documentation below on how to create an instance of ``Iou`` contract from the :doc:`Getting Started guide </app-dev/bindings-java/quickstart>`:
```
Comment (leo-da) [inline docs/source/json-api/index.rst:317]: @gerolf-da do we want to keep it as Quickstart guide for now?
Carol mentioned that `we are replacing Quickstart with Getting Started in SDK 1.0`...
changing the title to Getting Started, links does not change.

Comment (gerolf-da) [reply docs/source/json-api/index.rst:317]: Please keep the title as quickstart. We should continue to use unambiguous names between the two. Ideally the JSON API docs use templates and examples from the new getting started guide.

Comment (leo-da) [reply docs/source/json-api/index.rst:317]: reverted this part back to Quickstart guide

Comment (hurryabit) [reply docs/source/json-api/index.rst:317]: Shall we call it the "Java bindings quickstart guide" to be even more clear?

**Pull request merged**